### DISCUSSION TITLE
Feature request for GitHub Projects: display recent projects on the sidebar navigation, along with recent activities and repositories

### DISCUSSION THREAD

**Discussion by marcelovicentegc** [community/community#38165] [Other Feature Feedback, Questions, & Ideas]
Hey, all! 

The reason I'm opening this discussion: as a hard user of GitHub Projects, I would love to have a **Recent Projects** section along with the Recent Repositories and Recent Activities sections. That would make my life much easier, save me a few clicks, speed up work for others, and ease transitioning from one project managing software to GitHub Projects.

<img width="313" alt="Screen Shot 2022-11-03 at 09 50 49" src="https://user-images.githubusercontent.com/33183880/199729830-e736eccf-5715-47db-aa4a-9efa913a0feb.png">

> PS: Why not have a **Projects** category for discussions? 
> PS 2: It occurs to me that GitHub Projects is a takeover on top of other project managing tools and that more and more people will migrate from third-party tools to it, as my team and I are doing just that, coming from Jira.

Thanks 😉!

**Comment by ossanna16** [comment]
Hi there @marcelovicentegc and welcome to our community! Thank you for the feature request 🙂
